cmd/server/config/config.go

```diff
@@ -122,6 +92,19 @@ func LoadConfig(path string, v *Config) error {
 	return v.Validate()
 }
 
+func SaveConfig(path string, v *Config) error {
+	if v == nil {
+		return fmt.Errorf("config cannot be nil")
```

Comment:
```suggestion
		return fmt.Error("config cannot be nil")
```

Reply:
There is no `fmt.Error`, only `fmt.Errorf`

Reply:
`errors.New` perhaps?

Reply:
Good point!